tsql/ReadMe.md

```diff
@@ -4,4 +4,11 @@ This is a community supported grammar file for t-sql, it is cool, yet not very c
 so far because a grammar reference of T-SQL is hard to find.
 MS website has grammar reference for different statements,
 but it's not a complete file, so we try hard to stick to the references we could find,
-and bit by bit make it a more complete grammar.
\ No newline at end of file
+and bit by bit make it a more complete grammar.
+
+Use with Antlr 4.7
+------------------
+This grammar uses upper-case key-words only. That means, as sql is case-insensitive, that
+it can’t be used with Antlr 4.7 directly. You need a CharStream that converts to 
+upper-case for the lexer. An implementation for java can be found [on Github](https://github.com/antlr/antlr4/pull/2048/commits/8043e85d178a318172bc94639bd0c131d7e3c60c).
+
```

Comment:
There is a part of another PR. Is it possible to remove it?

Reply:
Sorry for that. I can rewrite the history of this branch if you prefer.

Reply:
It would be great.

Reply:
Force-pushed…